### ISSUE TITLE
Error: webdav timeout

### ISSUE THREAD

**Issue by sred36** [SqrTT/prophet#299]
Error: webdav timeout
Trying to re-upload file
Error: webdav timeout
Trying to re-upload file

**Comment by reejojoseph** [comment]
I, along with many, are facing this issue. @SqrTT could you please provide a workaround for this if possible?

**Comment by lyubok** [comment]
Same here. 

VS Code details:
Version: 1.59.0 (user setup)
Commit: 379476f0e13988d90fab105c5c19e7abc8b1dea8
Date: 2021-08-04T23:13:12.822Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Windows_NT x64 10.0.19043

**Comment by SqrTT** [comment]
Hi all, I can't provide workaround since seems this is networking or sandbox issue.

Could you, please, try to reach out sandbox via webdav in moment when face the issue in console?

**Comment by Reddy293** [comment]
```
Error: webdav timeout
Trying to re-upload cartridge


Error!?: webdav timeout
Error: webdav timeout
```

@SqrTT, we are able to connect sandbox via webdev but getting the above error while we uploading the code. and maximum it is happening in MacOS. could you please provide a workaround for this issue?
Thanks in advance.

**Comment by prgMgava** [comment]
I'm having the same problem.
A week ago it was working normally.
This is the log I get when I start prophet:
Error!?: webdav timeout
Error: webdav timeout
Trying to re-upload
Connection validated successfully
Current active version is: 'version1'
Start uploading cartridges
Cleanup code version...
Error: webdav timeout

I opened a ticket on salesforce to see if they can help me.
I can normally access the sandbox which is the same domain I use in prophet. Weird.
It's definitely a network problem. Because I changed the internet and it worked correctly. If I use my phone's 4g it doesn't reach 100mps it works but my 500mps broadband doesn't work.
